Modules/Devices/test/Devices.Domain.Tests/Tiers/TierCanBeDeleted.cs

```diff
@@ -30,7 +30,7 @@ public void Tier_with_related_identities_cannot_be_deleted()
 
         // Assert
         result.Should().NotBeNull();
-        result.Should().BeEquivalentTo(DomainErrors.CannotDeleteUsedTier(1));
+        result.Should().BeEquivalentTo(DomainErrors.CannotDeleteUsedTier($"The Tier is assigned to one or more Identities. A Tier cannot be deleted if there are Identities assigned to it ({1} found)."));
```

Comment:
```suggestion
        result.Should().BeEquivalentTo(DomainErrors.CannotDeleteUsedTier(""));
        result.Message.Should().Contain("Tier is assigned to one or more Identities");
```

Reply:
If we change this, wouldn't we get an error since the tier.CanBeDeleted call still passes a message argument and as such, the result will not be equivalent?
![image](https://github.com/nmshd/backbone/assets/118175875/b3dbde41-ab7c-49b0-9689-7f4c68bfb881)

Reply:
You're right. I thought we already had an Equals methods on the DomainError. I will add it.

Reply:
Done. When you now use

```csharp
result.Should().Be(DomainErrors.CannotDeleteUsedTier(""));
```
(BeEquivalentTo -> Be) 

it should work.